module-2/backend/week_one.md

```diff
@@ -5,28 +5,57 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+  Get: Get is used to access a page for read purposes
+  Post: Post is used for creating new data within an app
+  Delete: Delete is used to delete data within an app
+  Patch: Patch updates part of a resource, but not the whole thing. 
+  Put: updates a whole resource. 
 2. What is Sinatra?
+  Sinatra is a framework for making applications that incorporates Active Record and SQL databases. 
 4. What is MVC?
+  MVC stands for the Mode-View-Controller model for structuring applications
 5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
+  We use the RESTful route structure so that our applications are easy to read and understand according to a standardized structure.  
 6. What types of variables are accessible in our view templates without explicitly passing them?
+  Global Variables
 7. Given the following block of code, how would I pass an instance variable `count` with a value of `1` to my `index.erb` template?
   
   ```ruby
   get '/horses' do
+    @count = 1
+    @name = "Mr. Ed"
     erb :index
   end
   ```
 
 8. In the same code block, how would I pass a local variable `name` with a value of `Mr. Ed` to the view?
 9. What's the purpose of ERB?
+  Embedded Ruby files provide a place to integrate HTML and Ruby into one application view. 
 10. Why do I need a development AND test database?
+  So that you can a variety of tests without affecting the database you are using for production. 
 11. What is CRUD and why is it important?
+Create, Read, Update, Delete. These four things constitute the essential/basic function of a web app.
 12. What does HTTP stand for? 
+HyperText Transfer Protocol
 13. What are the two ways to interpolate Ruby in an ERB view template? What's the difference between these two ways?
+<%= %> or #{}

```

Comment:
looking for `<&= %>` or `<% %>`. one shows the information, the other allows us access to ruby